cincinnati/src/lib.rs

```diff
@@ -43,7 +43,7 @@ pub use std::collections::BTreeMap as MapImpl;
 pub use std::collections::BTreeSet as SetImpl;
 
 /// Graph type which stores `Release` as node-weights and `Empty` as edge-weights.
-#[derive(Debug, Default)]
+#[derive(Debug, Default, Clone)]
 #[cfg_attr(any(test, feature = "test"), derive(Clone))]
```

Comment:
needs to be removed so that `test` features would not attempt to re-implement Clone

Same for InternalIO:
```
#[derive(Debug, Clone)]
#[cfg_attr(test, derive(PartialEq))]
pub struct InternalIO {
    pub graph: cincinnati::Graph,
    pub parameters: HashMap<String, String>,
}
```